lib/actions/checkin.ts

```diff
@@ -5,7 +5,7 @@ import { encrypt, toCSV } from '../crypto'
 interface Params {
   ticket: api.TicketReq
   guest: db.Guest
-  companyId: api.CompanyRes['id']
+  companyId?: api.CompanyRes['id']
```

Comment:
Why is this optional now?

Reply:
So my thinking was that we do not save the proxy checkins, thus we do not need to add it to the guest in the db and if the companyId is `undefined` it just wont do anything.